client/src/containers/dashboard/Dashboard.js

```diff
@@ -1,10 +1,29 @@
 import React from "react";
+import { Link } from "react-router-dom";
+import { createSession } from "../../reducers/actions/index";
+import { useDispatch } from "react-redux";
 
 
 function Dashboard() {
+const dispatch = useDispatch()
+
+  const logout = ()=>{
+    dispatch(
+     createSession(
+   0,
+    '',
```

Comment:
It would it be easier to set an empty variable and pass that here instead of the multiple ```''```

Reply:
I will do, but this code will be overwritten by Seyma's